README.md

```diff
@@ -143,7 +150,25 @@ I would like to thank Microverse for this oppunity I get in improving myself pro
 <!-- COLLABORATORS -->
 
 ## 👥 Collaborators <a name="collaborators"></a>
-- GitHub: [https://github.com/NgachaIan]
+- GitHub: [https://github.com/Edgahkipkemoi]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
[Optional] Well! the FAQ section in your project's Readme file is optional. You have two options to consider regarding this section:

1. If you decide to keep the FAQ section:
   Please allocate some time to update it according to the specific needs and requirements of your project. Ensure that the questions and answers provided are relevant and helpful for users. Additionally, remember to remove the "optional" label from the Table of Contents as it is implied that the section is optional. 🙏 

2. If you choose to remove the FAQ section:
   If you believe the FAQ section is not necessary for your project, you can remove it completely from your Readme file. However, please remember to also remove it from the Table of Contents, ensuring consistency between the content and the navigational elements.

By either updating or removing the FAQ section, you can enhance the overall clarity and usability of your Readme file. If you have any questions or require further assistance, please feel free to ask. Keep up the good work! 👍